app/models/assignment.rb

```diff
@@ -95,4 +96,9 @@ def last_hour
   def delete_reminder
     Reminder.where(assignment: id).destroy_all
   end
+
+  def to_label
+    "#{self.class.name.humanize} - #{client.contact.full_name} - "\
```

Comment:
`contact` seems optional on the `Client`, so this could crash if you don't check for presence of the contact.

Reply:
where did you see that contact is optional for the client? 
because when a client is registered, the contact is required in the form.

Reply:
https://github.com/panter/aoz-003/blob/develop/app/models/contact.rb#L2

the relation is `optional: true`